README.md

```diff
@@ -136,15 +137,16 @@ To avoid specifying `tasmota_commands` for each host using host_vars you can use
 
 Sets the `TelePeriod` to 10 seconds to all devices specified in the `tasmota` host group.
 
-	- hosts: tasmota_devices  
+	- name: Setup tasmota devices with role
+      hosts: tasmota_devices  
 	  # disable fact gathering since this is currently not possible on tasmota devices  
-      gather_facts: no
+      ansible.builtin.gather_facts: false
       vars:
         tasmota_commands:
             - command: TelePeriod
               value: 10
 	  roles:
-	    - tobias_richter.tasmota
```

Comment:
The role name `tobias_richter.tasmota` is absolutely correct, see:  https://galaxy.ansible.com/ui/standalone/roles/tobias_richter/tasmota/

Ansible Galaxy rewrites github namespaces with for example "-" to "-" so `tobias-richter` is transformed to `tobias_richter`. So please revert this change.